addons/l10n_it_edi/views/l10n_it_view.xml

```diff
@@ -83,18 +83,6 @@
         </field>
     </record>
 
-    <record id="view_invoice_tree_inherit" model="ir.ui.view">
-        <field name="name">account.move.tree.inherit</field>
-        <field name="model">account.move</field>
-        <field name="inherit_id" ref="account.view_invoice_tree" />
-        <field name="arch" type="xml">
-            <field name="state" position="before">
-                <field name="l10n_it_edi_transaction" optional="hide"/>
-                <field name="l10n_it_edi_attachment_id" optional="hide"/>
-            </field>
-        </field>
-    </record>
-
```

Comment:
Guaps. This made the fields appear duplicated, as the fields appeared in two levels of inheritance for the same view. My bad on the other PR.